paddle/scripts/travis/check_style.sh

```diff
@@ -13,7 +13,7 @@ export PATH=/usr/bin:$PATH
 pre-commit install
 clang-format --version
 
-if ! pre-commit run -a ; then
```

Comment:
Why no `-a`? Travis don't know PR's diffs, it only fetch the corresponding branch